consensus/consensus.go

```diff
@@ -42,6 +42,11 @@ type Consensus struct {
 	// channel to receive consensus message
 	MsgChan chan []byte
 
+	// whether this node belong to genesis accounts
+	isHarmonyAccount bool
```

Comment:
Please retire `isHarmonyAccount` logic altogether, now that we have `delayCommit`.